inc/ConfigMap.h

```diff
@@ -97,6 +116,10 @@ namespace PCOE {
          */
         static void addSearchPath(const std::string& path);
 
+        unsigned long size();
```

Comment:
Remove `size` and `insert`. Neither is an inherent characteristic of the class, so they shouldn't be part of the public API.